skycultures/arabic/constellation_names.eng.fab

```diff
@@ -0,0 +1,138 @@
+0300 "بنات نعش" _("Daughters Of Na'sh")
+0400 "الظباء وأولاد الظباء" _("The Gazelles And Thier Youngsters")
+0410 "القفزة الأولى" _("The First Leap")
+0420 "القفزة الثانية" _("The Second Leap")
+0430 "القفزة الثالثة" _("The Third Leap")
+0501 "الفارطان" _("The Two Preceding To The Water")
+0500 "الحوض" _("The Pool")
+0600 "الذئبان" _("The Two Wolves")
+0601 "أظفار الذئب" _("The Wolf Claws")
+0700 "العوائذ" _("The Mother Camels")
+0800 "الأثافي" _("The Three Stone Support")
+0900 "القدر" _("The Pot")
+1000 "الراعي الشمالي والشاء" _("The Shepherd And The Sheep")
+1001 "كوكبا الفرق" _("The Two Separated Stars")
+1100 "ذو السلاح" _("The Armed One")
+1101 "الرمح" _("The Spear")
+1200 "السماك الأعزل" _("The Unarmed One")
+1300 "الضباع وأولاد الضباع" _("The Hyenas And Their Youngsters")
+#3 page
+1400 "قصعة المساكين" _("The Bowl Of The Poor")
+1500 "الروضة والأغنام" _("The Desert Garden And The Goats")
+1501 "النسق الشآمي" _("The Northern Line")
+1502 "النسق اليماني" _("اthe Southern Line")
+1600 "النسر الواقع" _("The Landing Eagle")
+1700 "النسر الطائر" _("The Flying Eagle")
+1800 "الفوارس" _("The Knights")
+1900 "الناقة" _("The She-Camel")
+1901 "رأس الناقة" _("Head Of Tthe She-Came")
+1902 "عنق الناقة" _("Neck Of Tthe She-Came")
+1903 "يدا الناقة" _("Legs Of Tthe She-Came")
+#The name is left blank intentionally, label of 2604 Al-Thurayya is sufficient. This constellation is to show the complete Al-Thurayya image with its two hands.
+#2000 "الثريا " _("Al-Thurayya")
+2000 " " _("")
+2001 "سنام الناقة - الكف الخضيب" _("The She-Camel's Hump - The Henna Tinted Hand")
+2002 "العاتق" _("The Shoulder Blade")
+2003 "المنكب" _("The Shoulder")
+2004 "العضد" _("The Arm")
+2005 "المرفق" _("The Elbow")
+2006 "الساعد" _("The Forearm")
+2007 "المعصم" _("The Wrist")
+2008 "الكف الجذماء" _("The Leprous Hand")
+#3 page
+# TRANSLATORS: Al-Ayyuq is a proper name meaning the obstructor
+2100 "العيوق والأعلام" _("Al-Ayyuq And The Posts")
+2200 "الخباء الشآمي" _("The Northern Tent")
+2300 "القعود" _("The Young Camel")
+2400 "عناق الأرض" _("The Caracal")
+2500 "الدلو" _("The Well Bucket")
+2501 "الكرب" _("The Bucket Rope Knot")
+2502 "الفرغ المقدم" _("The Front Bucket Mouth")
+2504 "الفرغ المؤخر" _("The Rear Bucket Mouth")
+2506 "الرشاء" _("The Bucket Rope")
+#For reference 2 see vol. 1 pg.
+2550 "الحوت" _("The Fish")
+2560 "بلدة الثعلب" _("The Empty Place Of The Fox")
+2600 "الحمل" _("The Lamb")
+#This name has been corrupted in many nanuscripts and the printed references from the correct "الأيبسان" to "الأنيسان". I discussed the name in my book Reference 4.
+2601 "الأيبسان" _("The Two Forelegs Of The Lamb")
+2602 "الشرطان" _("The Two Signs")
+2603 "البطين" _("The Little Abdomen")
+# TRANSLATORS: Al-Thurayya is a proper name
+2604 "الثريا" _("Al-Thurayya")
+#This constellation defines the boundary of Al-Dabaran, his camels and two dogs. Name is intentionally left blank
+#2700 "الدبران وقلاصه وكلباه" _("Al-Dabaran, His Camels And Two Dogs")
+#2700 " " _(" ")

```

Comment:
Why it empty?

Reply:
This one is commented out.
It is kept for reference